src/content/en/updates/2018/06/ar-for-the-web.md

```diff
@@ -0,0 +1,277 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Augmented reality allows placement and tracking of virtual objects in a real-world view.
+
+{# wf_updated_on: 2018-06-15 #}
+{# wf_published_on: 2018-06-14 #}
+{# wf_tags: immersive-web,webar #}
+{# wf_blink_components: Blink>WebVR #}
+{# wf_featured_image: /web/updates/images/2018/06/sunflowers.png #}
+{# wf_featured_snippet: Augmented reality allows placement and tracking of virtual objects in a real-world view. #}
+
+# Augmented reality for the web {: .page-title }
```

Comment:
Title Caps for Titles?

Reply:
I thought DevSite titles were sentence case now.

Reply:
OK, haven't been enforcing that. Oops.